proprietary/tokens/src/components/ams/table-of-contents.tokens.json

```diff
@@ -0,0 +1,56 @@
+{
+  "ams": {
+    "table-of-contents": {
+      "color": { "value": "{ams.color.primary-black}" },
```

Comment:
Can you check if all tokens are used in the CSS? At a quick glance, they aren't

Reply:
Checked